identity_iota_core/Cargo.toml

```diff
@@ -46,9 +51,9 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 default = ["client", "iota-client", "revocation-bitmap", "send-sync-client-ext"]
 # Exposes the `IotaIdentityClient` and `IotaIdentityClientExt` traits.
-client = ["dep:async-trait", "dep:iota-types"]
+client = ["dep:async-trait", "dep:iota-sdk"]
 # Enables the iota-client dependency, the client trait implementations for it, and the `IotaClientExt` trait.
-iota-client = ["dep:iota-client", "client"]
+iota-client = ["dep:iota-sdk", "client"]
```

Comment:
Since `iota-sdk` is activated whenever `client` is activated, we don't need to include `dep:iota-sdk` in the `iota-client` feature.